src/test/java/com/marizueva/laboratory/hw2/ex2/LoginTestWithSoftAssertions.java

```diff
@@ -0,0 +1,243 @@
+        softAssert.assertTrue(subMenuFromLeftComplexTableItem.isDisplayed());
+        WebElement subMenuFromLeftUserTableItem = driver.findElement(
+                By.xpath("//div[@name='navigation-sidebar']//a[@href='user-table.html']"));
+        softAssert.assertTrue(subMenuFromLeftUserTableItem.isDisplayed());
+        WebElement subMenuFromLeftTableWithPagesItem = driver.findElement(
+                By.xpath("//div[@name='navigation-sidebar']"
+                        + "//a[@href='table-pages.html']"));
+        softAssert.assertTrue(subMenuFromLeftTableWithPagesItem.isDisplayed());
+        WebElement subMenuFromLeftDifferentElementsItem = driver.findElement(
+                By.xpath("//div[@name='navigation-sidebar']"
+                        + "//a[@href='different-elements.html']"));
+        softAssert.assertTrue(subMenuFromLeftDifferentElementsItem.isDisplayed());
+
+        // Open through the header menu Service -> Different Elements Page
+        headerItemService.click();
+        subMenuFromHeaderDifferentElementsItem.click();
+
+        //Check interface on Different elements page, it contains all needed
+        // elements 4 checkboxes, 4 radios, 1 dropdown, 2 buttons
+
+        List<WebElement> checkBoxes = driver.findElements(
+                By.cssSelector("input[type='checkbox'"));
+        softAssert.assertEquals(checkBoxes.size(), 4);
+
+        List<WebElement> radioButtons = driver.findElements(
+                By.cssSelector("input[type='radio'"));
+        softAssert.assertEquals(radioButtons.size(), 4);
+
+        List<WebElement> dropdownMenu = driver.findElements(
+                By.cssSelector("div.colors"));
+        softAssert.assertEquals(dropdownMenu.size(), 1);
+
+        // Assert that there is Right Section
+        WebElement rightPanel = driver.findElement(
+                By.cssSelector(".uui-side-bar.right-fix-panel"));
+        softAssert.assertTrue(rightPanel.isDisplayed());
+
+        // Assert that there is Left Section
+        WebElement leftPanel = driver.findElement(By.id("mCSB_1_container"));
+        softAssert.assertTrue(leftPanel.isDisplayed());
+
+        // Select checkboxes Water, Wind
+        WebElement waterCheckbox = driver
+                .findElement(By.xpath("//label[@class='label-checkbox'][1]"));
+        waterCheckbox.click();
+        WebElement windCheckbox = driver
+                .findElement(By.xpath("//label[@class='label-checkbox'][3]"));
+        windCheckbox.click();
+
+        // Assert that for each checkbox there is an individual log row
+        // and value is corresponded to the status of checkbox.
+        // Log rows are displayed, checkbox name and its status is corresponding to selected
+        WebElement waterLogTrue = driver.findElement(By
+                .xpath("//*[contains(text(),"
+                        + "substring('01:14:22  Water: condition changed to true',11, 16))]"));
```

Comment:
Looks like that similar comments I've already put in the hw3 review. could you please replace locator without using text